app/js/components/labOrderEntry/LabEntryForm.jsx

```diff
@@ -14,46 +18,54 @@ export class LabEntryForm extends React.Component {
     categoryId: 1,
     defaultTests: [],
     selectedTests: [],
-    selectedPanel: null,
+    selectedPanelIds: [],
     disableSaveButton: true,
     disableCancelButton: true,
   };
 
-  selectTest = (testId) => {
-    if (!this.state.selectedTests.includes(testId) && !this.state.defaultTests.includes(testId)) {
+  selectTest = (test) => {
+    const { addDraftLabOrdersToStore, deleteDraftLabOrderFromStore } = this.props;
+    const isSelectedIds = this.state.selectedTests.map(currentTest => currentTest.id);
```

Comment:
so usually the prefix `is` normally used to depict booleans so try a better naming convention, something like `selectedIds`

Reply:
You told me to use isSelectedIds

Reply:
done